arrays.md

```diff
@@ -15,11 +15,13 @@ section on [Arrays and vectors](data-structures.html).
 
 Some libraries available on [Quicklisp](https://www.quicklisp.org/beta/) for manipulating arrays:
 
-- [array-operations](https://github.com/tpapp/array-operations) defines
+- [array-operations](https://github.com/Lisp-Stat/array-operations) maintained by @Symbolics defines
   functions `generate`, `permute`, `displace`, `flatten`, `split`,
   `combine`, `reshape`. It also defines `each`, for element-wise
-  operations. This library is not maintained by the original author,
-  but there is an [actively maintained fork](https://github.com/bendudson/array-operations).
+  operations. This was a fork of [bendudson/array-operations on
```

Comment:
This should probably read: 'this _is_ a fork ...'

Reply:
Ok, I changed it to present tense.